src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,72 @@
 package gr11review.part2;
 
+import java.io.*;
+import java.io.FileNotFoundException;
+
 public class Utility {
-    
-}
\ No newline at end of file
+  public static String zipZap(String strWord){
+
+    int intCount;
+    String strNewWord;
+
+    strNewWord = "";
+
+    if(strWord.length() <= 2){
+      return strWord;
+    }
+
+    for(intCount = 0; intCount < strWord.length(); intCount++){
+      if(strWord.substring(intCount, intCount + 1).equalsIgnoreCase("z") && strWord.substring(intCount + 2, intCount + 3).equalsIgnoreCase("p")){
+        strNewWord = strNewWord + strWord.substring(intCount, intCount + 1) + strWord.substring(intCount + 2, intCount + 3);
+        intCount = intCount + 2;
+      }else{
+        strNewWord = strNewWord + strWord.substring(intCount, intCount + 1);
+      }
+    }
+
+    return strNewWord;
+
+  }
+
+  public static String longestword(String filenametxt) throws FileNotFoundException, IOException{
+
+    File filename = new File("src/gr11review/part2/words.txt");
+
+    BufferedReader keyboard = new BufferedReader(new FileReader(filename));
```

Comment:
filename is provided as a parameter to the method, should not be hard coded in to the program